openedx/core/djangoapps/course_roles/permissions.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
A lot of apps define a permissions API to define permissions helpers for their own apps, but given that this is the centralized store of permissions to query, I think it makes more sense as a `data.py` per OEP-49 conventions.